src/gt4py/next/iterator/transforms/pass_manager.py

```diff
@@ -82,15 +82,15 @@ def apply_common_transforms(
     unconditionally_collapse_tuples=False,
 ):
     if lift_mode is None:
-        lift_mode = LiftMode.FORCE_INLINE
+        lift_mode = LiftMode.FORCE_TEMPORARIES
```

Comment:
```suggestion
        lift_mode = LiftMode.FORCE_INLINE
```
This was only meant as a test to see how the pipeline performs.